src/js/stores/managers/layout-manager.js

```diff
@@ -21,8 +22,7 @@ class LayoutManager {
         var loadedLayout = this.getLayoutFromLocalStorage();
 
         if(loadedLayout.length === 0) {
-            this.addComponent(LayoutComponentConstants.SceneList);
-            this.addComponent(LayoutComponentConstants.SceneMediaBrowser);
+            this.layout = this.loadPreset(PresetLayouts.default)
```

Comment:
Looking good, I'd expect a unit test covering the loading of a preset if no layout is found in local storage.

We might need to consider validating layouts loaded from local storage.  If a user has used a previous version of the software, their local storage saved layout might include a component (eg in this case a Scene List) that we do not offer anymore.

Reply:
Ok, would a simple check of LayoutManager.Layout === PresetLayouts.default while striping out the ID key be sensible?

Reply:
Agreed the loading from local storage needs validation. It appears that any invalid components get striped from observation but haven't looked at why and probably needs formalizing.

Reply:
Yep I think just updating the unit test "should setup a basic layout" can do your equality check and all shall be good.  As mentioned after update, your validation of local storage layout looks great.